webapp/lib/app/nook/view.dart

```diff
@@ -1798,17 +1798,22 @@ class TagActionView implements ActionView {
       ..text = shortcut;
     action.append(_shortcutElement);
 
-    var tagElement = TagView(text, "");
+    var tagElement = TagView(text, "")
+      ..onSelect = _addTag;
     action.append(tagElement.renderElement);
 
     _buttonElement = new DivElement()
       ..classes.add('action__button')
       ..classes.add('action__button--float')
       ..text = buttonText
-      ..onClick.listen((_) => _view.appController.command(UIAction.addTag, new TagData(action.dataset['id'])));
+      ..onClick.listen((_) => _addTag());
     action.append(_buttonElement);
   }
 
+  void _addTag() {
```

Comment:
nit - suggest renaming this to `_addTagCommandCallback`

Reply:
Fixed!